markets/perps-market/test/integration/helpers/fillPrice.ts

```diff
@@ -0,0 +1,19 @@
+import Wei, { wei } from '@synthetixio/wei';
+
+// Calculates PD
+const calculatePD = (skew: Wei, skewScale: Wei) => wei(skew).div(wei(skewScale));
+// Calculates the price with pd applied
+const calculateAdjustedPrice = (price: Wei, pd: Wei) => wei(price).add(wei(price).mul(pd));
```

Comment:
price is already wei so can prolly remove the extra one

Reply:
Thanks!